src/microbe_stage/Microbe.cs

```diff
@@ -518,9 +533,15 @@ public void Damage(float amount, string source)
         }
         else if (source == "chunk")
         {
+            PlaySoundEffect("res://assets/sounds/soundeffects/microbe-toxin-damage.ogg");
```

Comment:
There should be a todo comment here that, this should play an ice damage sound effect as that's where this damage comes from.

Reply:
I don't see you adding a comment here?